server/cluster/metadata/topology_manager.go

```diff
@@ -303,9 +303,10 @@ func (m *TopologyManagerImpl) EvictTable(ctx context.Context, tableID storage.Ta
 		}
 
 		// Update shardView in storage.
+		newShardView := storage.NewShardView(shardView.ShardID, prevVersion+1, tableIDs)
```

Comment:
I guess the version should be set by the outside, because it should be confirmed by ceresdb node. Maybe we leave a TODO for it.